### DISCUSSION TITLE
记人生第一次徒步 30 公里 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#53] [Announcements]
# 记人生第一次徒步 30 公里 - forecho's Blog

徒步前
刚听到公司组织这个活动的消息时，我内心是第一反应是拒绝的，心想公司就会搞这种穷 B 的活动，但是随后又是兴奋的。
去年老板就在早会有跟我们分享他去徒步戈壁滩挑战（听说是徒步 4 天）的经历，说是有机会组织一次这样的活动，以前我对 30 公里真没什么概念，那天翻了翻手机才发现我平常上下班也就最多走 7 公里，真不知道徒步 30 公里是什么体验，所以还真想借此机会测试和挑战一下自己。
徒步前有同事分享一些注意事项，我记住的大致有这些：

https://blog.forecho.com/life-is-30km-on-foot-for-the-first-time.html

**Comment by dox-xu** [comment]
最近也打算长距离city walk，感谢你分享的经验

**Reply by dox-xu** [reply]
前几天才走，也是30公里
https://post.smzdm.com/p/al986k70/
